framework/src/bundle/BundlePrivate.cpp

```diff
@@ -64,15 +64,15 @@ void BundlePrivate::Stop(uint32_t options)
   std::exception_ptr savedException;
 
   {
-    auto l = coreCtx->resolver.Lock();
+    auto l = this->Lock();
 
     // 1:
     if (state == Bundle::STATE_UNINSTALLED) {
       throw std::logic_error("Bundle is uninstalled");
     }
 
     // 2: If an operation is in progress, wait a little
-    WaitOnOperation(coreCtx->resolver, l, "Bundle::Stop", false);
+    // Note: This step hass been removed with removal of BundleThread
```

Comment:
if BundleThreads is being removed, it doesn't make sense to add comments that reference it. I'd suggest updating the comments as if BundleThreads doesn't exist.